uber/models/attendee.py

```diff
@@ -869,6 +873,10 @@ def normalized_email(cls):
     def normalize_email(cls, email):
         return email.strip().lower().replace('.', '')
 
+    @property
+    def gets_emails(self):
```

Comment:
This finally lets us guarantee that no scheduled automated emails will be sent to badges that are not either New or Complete.

Reply:
Makes sense.  I *think* that when we send emails to e.g. people who have just transferred a badge it's not through the scheduled automated email process, so this shouldn't interfere with that.